db/view/view_update_backlog.hh

```diff
@@ -23,12 +23,27 @@ namespace db::view {
  * backlog - how much view hints are consuming. The size of a backlog is relative
  * to its maximum size.
  */
-struct update_backlog {
-    size_t current;
-    size_t max;
+class update_backlog {
+    size_t _current;
+    size_t _max;
```

Comment:
Can't they just be `const size_t`? I don't think there would be a need for getters if you did that. You wouldn't have to hide the fields and rename them. All that will lead to a smaller diff.

Reply:
I was considering whether maybe in the future we will want to also add setters for these fields (with some checks), but we probably won't so for now the approach with `const size_t` does seem better